yellowbrick/features/pca.py

```diff
@@ -453,7 +545,7 @@ def pca_decomposition(
 
     # Fit and transform the visualizer (calls draw)
     visualizer.fit(X, y)
-    visualizer.transform(X)
+    visualizer.transform(X, y)
     visualizer.poof()
 
     # Return the axes object on the visualizer
```

Comment:
Instead of returning the axes, please return the visualizer!

Reply:
Sure!